### PR TITLE
financial data changes 

### PR DESCRIPTION
**Pull request by dreamfall**

### CODE REVIEW

File: app/form_pointers/form_financial_pointer.rb [hunk 1]
```diff
@@ -59,6 +63,20 @@ def growth_overseas_earnings(year)
     end
   end
 
+  def growth_in_total_turnover(year)
```
Comment (matthewford) [inline app/form_pointers/form_financial_pointer.rb]: Assignment Branch Condition size for growth_in_total_turnover is too high. [21.61/15]

File: app/form_pointers/uk_sales_calculator.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class UkSalesCalculator
+  attr_reader :fetched_data
+
+  def initialize(fetched_data)
+    @fetched_data = fetched_data
+  end
+
+  def data
```
Comment (matthewford) [inline app/form_pointers/uk_sales_calculator.rb]: Assignment Branch Condition size for data is too high. [16.88/15]

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,149 @@
+require "rails_helper"
+
+describe FormFinancialPointer do
+  let(:trade_data) do
+    [
+     {
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+       employees: [
+                   { value: "1", name: "employees_1of6" },
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+                   { value: "2", name: "employees_2of6" },
+                   { value: "3", name: "employees_3of6" },
+                   { value: "4", name: "employees_4of6" },
+                   { value: "5", name: "employees_5of6" },
+                   { value: "6", name: "employees_6of6" }
+                  ]
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Indent the right bracket the same as the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+     },
+     {
+       overseas_sales: [
+                        { value: "30", name: "overseas_sales_1of6" },
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+                        { value: "40", name: "overseas_sales_2of6" },
+                        { value: "50", name: "overseas_sales_3of6" },
+                        { value: "60", name: "overseas_sales_4of6" },
+                        { value: "70", name: "overseas_sales_5of6" },
+                        { value: "80", name: "overseas_sales_6of6" }
+                       ]
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Indent the right bracket the same as the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+     },
+     {
+       overseas_sales_direct: [
+                               { value: "20", name: "overseas_sales_direct_1of6" },
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+                               { value: "30", name: "overseas_sales_direct_2of6" },
+                               { value: "40", name: "overseas_sales_direct_3of6" },
+                               { value: "50", name: "overseas_sales_direct_4of6" },
+                               { value: "60", name: "overseas_sales_direct_5of6" },
+                               { value: "70", name: "overseas_sales_direct_6of6" }
+                              ]
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Indent the right bracket the same as the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+     },
+     {
+       overseas_sales_indirect: [
+                                 { value: "10", name: "overseas_sales_indirect_1of6" },
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+                                 { value: "10", name: "overseas_sales_indirect_2of6" },
+                                 { value: "10", name: "overseas_sales_indirect_3of6" },
+                                 { value: "10", name: "overseas_sales_indirect_4of6" },
+                                 { value: "10", name: "overseas_sales_indirect_5of6" },
+                                 { value: "10", name: "overseas_sales_indirect_6of6" }
+                                ]
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Indent the right bracket the same as the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+     },
+     {
+       total_turnover: [
+                        { value: "100", name: "total_turnover_1of6" },
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 13]
```diff
@@ -0,0 +1,149 @@
...
+                        { value: "120", name: "total_turnover_2of6" },
+                        { value: "120", name: "total_turnover_3of6" },
+                        { value: "130", name: "total_turnover_4of6" },
+                        { value: "120", name: "total_turnover_5of6" },
+                        { value: "150", name: "total_turnover_6of6" }
+                       ]
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Indent the right bracket the same as the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+     },
+     {
+       net_profit: [
+                    { value: "10", name: "net_profit_1of6" },
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 15]
```diff
@@ -0,0 +1,149 @@
...
+                    { value: "10", name: "net_profit_2of6" },
+                    { value: "15", name: "net_profit_3of6" },
+                    { value: "15", name: "net_profit_4of6" },
+                    { value: "20", name: "net_profit_5of6" },
+                    { value: "30", name: "net_profit_6of6" }
+                   ]
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Indent the right bracket the same as the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 16]
```diff
@@ -0,0 +1,149 @@
...
+     },
+     {
+       total_imported_cost: [
+                             { value: "80", name: "total_imported_cost_1of6" },
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+                             { value: "80", name: "total_imported_cost_2of6" },
+                             { value: "90", name: "total_imported_cost_3of6" },
+                             { value: "100", name: "total_imported_cost_4of6" },
+                             { value: "100", name: "total_imported_cost_5of6" },
+                             { value: "120", name: "total_imported_cost_6of6" }
+                            ]
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Indent the right bracket the same as the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 18]
```diff
@@ -0,0 +1,149 @@
...
+     }
+    ]
+  end
+
+  let(:innovation_data) do
+    [
+     { employees: [
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 19]
```diff
@@ -0,0 +1,149 @@
...
+                   { value: "10", name: "employees_1of5" },
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 20]
```diff
@@ -0,0 +1,149 @@
...
+                   { value: "12", name: "employees_2of5" },
+                   { value: "12", name: "employees_3of5" },
+                   { value: "15", name: "employees_4of5" },
+                   { value: "18", name: "employees_5of5" }
+                  ]
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Indent the right bracket the same as the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 21]
```diff
@@ -0,0 +1,149 @@
...
+      { employees: [
+                     { value: "10", name: "employees_1of5" },
+                     { value: "12", name: "employees_2of5" },
+                     { value: "12", name: "employees_3of5" },
+                     { value: "15", name: "employees_4of5" },
+                     { value: "18", name: "employees_5of5" }
+                   ]
+      },
+      { total_turnover: [
+                          { value: "1000", name: "total_turnover_1of5" },
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.

File: spec/form_pointers/form_financial_pointer_spec.rb [hunk 22]
```diff
@@ -0,0 +1,149 @@
...
+                          { value: "1100", name: "total_turnover_2of5" },
+                          { value: "1200", name: "total_turnover_3of5" },
+                          { value: "1300", name: "total_turnover_4of5" },
+                          { value: "1300", name: "total_turnover_5of5" }
+                        ]
```
Comment (matthewford) [inline spec/form_pointers/form_financial_pointer_spec.rb]: Indent the right bracket the same as the start of the line where the left bracket is.

File: app/form_pointers/uk_sales_calculator.rb (hunk 2 above)
Comment (matthewford) [inline app/form_pointers/uk_sales_calculator.rb]: Assignment Branch Condition size for data is too high. [19.1/15]

**Comment by RuslanHamidullin** [comment]
Good to merge :+1:

**Pull request merged**